lib/Doctrine/Migrations/Generator/DefaultFileBuilder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Generator;
+
+use DateTimeImmutable;
+use DateTimeInterface;
+use function sprintf;
+
+/**
+ * The DefaultFileBuilder class is responsible for building a migration SQL file from an array of queries per version.
+ *
+ * @internal
+ */
+final class DefaultFileBuilder implements FileBuilder
```

Comment:
Using "default" as prefix isn't quite clear. Perhaps `ConcatenationFileBuilder` or something similar would be more explicit.

Reply:
You are right. Your suggestion is much better.
I'm also not happy with the other "default" named classes, but could not get better ideas...